src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -347,7 +347,7 @@ def _configure_db_dw_params(arg_ctx):
     arg_ctx.argument('read_scale',
                      arg_type=read_scale_param_type)
 
-    arg_ctx.argument('read_replica_count',
+    arg_ctx.argument('high_availability_replica_count',
                      arg_type=read_replicas_param_type)
```

Comment:
Should Line537 be renamed too? https://github.com/Azure/azure-cli/blob/6ab0ebf4b419ca191ded239f6e93bdada86816d3/src/azure-cli/azure/cli/command_modules/sql/_params.py#L537-L539